server/app/crud.py

```diff
@@ -137,7 +142,7 @@ def add_vector(
             ]
         )
 
-        if vector.level > 0:
+        if vector.level == 2:
```

Comment:
Can you use explicit naming please: `VectorLevel.LEVEL1`

Reply:
This has already been solved.

Reply:
Sure?

Reply:
Would it be OK to change it to:
```python
if vector.level not in [VectorLevel.BACKBONE, VectorLevel.LEVEL0]:
```
That way it will also work for Level 2 constructs.

Reply:
Is that the intention? Because it seems the logic was changed *away* from that. Do you know why this was changed?

Reply:
I can't remember why.
I've pushed the fixed in the last commit.